### ISSUE TITLE
Audio Track Names Do Not Load Correctly When Switching Audio Profiles in v2.42

### ISSUE THREAD

**Issue by yammes08** [staxrip/staxrip#1487]
**Describe the bug**
When switching audio profiles, defined audio track names are not loaded correctly.

**Expected behavior**
When switching audio profiles, defined audio track names are loaded correctly.

**How to reproduce the issue**
1. Create an audio profile, and give the track a name - eg `E-AC-3 5.1`
2. Create a second audio profile, and give the audio track a name - eg `AAC 1.0`
3. Switch between the profiles, then press edit, and observe that the incorrect audio track name is often loaded.

**Provide information**
- StaxRip x2.42.1

**Comment by Dendraspis** [comment]
That is sort of by design and intended.

What do you try to achieve? I assume you want to select a profile and overwrite whatever the Track Name is/was before?

**Comment by yammes08** [comment]
> That is sort of by design and intended.
> 
> What do you try to achieve? I assume you want to select a profile and overwrite whatever the Track Name is/was before?

I don't understand why this would be intended?

The reason I was looking was just to confirm I'd set my profiles up correctly. I'll give some more detail on the steps I took.

First, I created a profile called `AAC 1.0`. Within that profile, I set the track name to `AAC 1.0`. When I loaded that profile, and clicked `Edit` next to the profile name, the audio track name was blank. So, thinking I'd forgotten to add it, I added the name here.

Next, I made a new profile called `E-AC-3 5.1`. Here's a screenshot of it, and I'm viewing this by clicking `Edit Profiles...`:

![Screenshot 2024-09-16 002739](https://github.com/user-attachments/assets/f7d6f0cb-2530-484e-9b53-d55a846813f5)
Note the track name is `E-AC-3 5.1`.

Now, if I load that profile, then click `Edit` next to the name of the profile, I get:
![Screenshot 2024-09-16 002821](https://github.com/user-attachments/assets/9fe86551-f2f8-4001-afd2-205ce6102836)
The track name is incorrect.

In short, when I load a profile, I would expect the profile's audio track name to also be loaded. However, it isn't being loaded.

**Comment by jessielw** [comment]
> That is sort of by design and intended.
> 
> What do you try to achieve? I assume you want to select a profile and overwrite whatever the Track Name is/was before?

The old behavior was that your audio track profiles kept the defined name, this would be expected.

**Comment by Dendraspis** [comment]
Currently the entered `Track Name` is kept when you change the Audio Profile, so in case you enter a track name or it is read out via the file name, it is not overwritten when you decide you want to use another profile. 
This is the part that is intended, but I can see why you would like to be able to overwrite it....

**Comment by yammes08** [comment]
> Currently the entered `Track Name` is kept when you change the Audio Profile, so in case you enter a track name or it is read out via the file name, it is not overwritten when you decide you want to use another profile. This is the part that is intended, but I can see why you would like to be able to overwrite it....

Yes - my saved audio profiles also contain the track name, so I would want the track name to be loaded when the profile is loaded. This is already the behaviour in 2.41.X.

**Comment by Dendraspis** [comment]
I will implement a solution so you can get the behavior you would like to have for a future release, but it might take some time for the release...

**Comment by yammes08** [comment]
Nice, thank you!
